Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/handler/PlayerSyncMessageHandler.java

```diff
@@ -0,0 +1,33 @@
+package nl.soccar.gameserver.controller.socnet.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import nl.soccar.gameserver.controller.socnet.message.PlayerSyncMessage;
+import nl.soccar.socnet.connection.Connection;
+import nl.soccar.socnet.message.MessageHandler;
+
+public final class PlayerSyncMessageHandler extends MessageHandler<PlayerSyncMessage> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)